packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -77,7 +77,7 @@ module.exports = {
   mode: 'development',
   // You may want 'eval' instead if you prefer to see the compiled output in DevTools.
   // See the discussion in https://github.com/facebook/create-react-app/issues/343
-  devtool: 'cheap-module-source-map',
+  devtool: 'source-map',
```

Comment:
It's tough. I'd be curious to get thoughts from others. In my experience, `source-map` gets pretty slow for large bundles because it is a lot of work to regenerate the full map of the bundle. You can't cache things very well because any change to any file will force the recalculation of mappings for any file after the one that changed in the bundle. I usually go for `eval-source-map` for that reason.